src/parser/jobs/ast/ArcanaGroups.js

```diff
@@ -82,6 +82,37 @@ export const SPIRE_USED = [
 	ACTIONS.THE_SPIRE_FROM_SPREAD.id,
 ]
 
+/* utility */
+export const HELD_ARCANA_USE = [
+	ACTIONS.THE_BALANCE_FROM_SPREAD.id,
+	ACTIONS.THE_BOLE_FROM_SPREAD.id,
+	ACTIONS.THE_ARROW_FROM_SPREAD.id,
+	ACTIONS.THE_SPEAR_FROM_SPREAD.id,
+	ACTIONS.THE_EWER_FROM_SPREAD.id,
+	ACTIONS.THE_SPIRE_FROM_SPREAD.id,
+	ACTIONS.THE_BALANCE_EXPANDED_FROM_SPREAD.id,
+	ACTIONS.THE_BOLE_EXPANDED_FROM_SPREAD.id,
+	ACTIONS.THE_ARROW_EXPANDED_FROM_SPREAD.id,
+	ACTIONS.THE_SPEAR_EXPANDED_FROM_SPREAD.id,
+	ACTIONS.THE_EWER_EXPANDED_FROM_SPREAD.id,
+	ACTIONS.THE_SPIRE_EXPANDED_FROM_SPREAD.id,
+]
+
+export const DRAWN_ARCANA_USE = [
```

Comment:
DRAWN_ARCANA_USE is defined twice.

Reply:
You might be getting cross-eyed, there's DRAWN_ARCANA_USE which encompasses the actions, and DRAWN_ARCANA which are the Drawn statuses. If you can think of a better name for these variables I'll change them

Reply:
I'm sorry I should have been clearer. What I meant is that there is DRAWN_CARD_USE, which I added, and DRAWN_ARCANA_USE, which you added, that both encompass the same cards. We only want one name. I think your name likely better fits the rest of the group, but that means the name in my module would need to be changed.

Regardless there should only be one.

Reply:
Oh I see. Totally missed that! I'll delete the extra then. Would you like me to change your module var names in this commit too?

Reply:
Yes that would be great thank you!